pkg/strategy/controller_test.go

```diff
@@ -157,12 +157,21 @@ func TestSetSecurityToOAuthProxyByDefaultOnOpenShift(t *testing.T) {
 	assert.Equal(t, v1.IngressSecurityOAuthProxy, jaeger.Spec.Ingress.Security)
 }
 
-func TestSetSecurityToNoneOnNonOpenShift(t *testing.T) {
+func TestSetSecurityOnNonOpenShift(t *testing.T) {
```

Comment:
This is not dependent on being on OpenShift anymore, so, this can be renamed, `TestSetExplicitSecurity` could be appropriate.